terraform/modules/database/main.tf

```diff
@@ -0,0 +1,23 @@
+resource "aws_db_instance" "api3-operations" {
+  identifier             = "api3-operations"
+  instance_class         = "db.t3.micro"
+  allocated_storage      = 5
+  engine                 = "postgres"
+  engine_version         = "13.1"
+  username               = var.db_username
+  password               = var.db_password
+  db_subnet_group_name   = aws_db_subnet_group.api3-operations.name
+  vpc_security_group_ids = [aws_security_group.rds.id]
+  publicly_accessible    = false
+  skip_final_snapshot    = true
```

Comment:
It'd be better to be available on multiple zones:
> multi_az               = true

Also we can enable to cloudwatch logs to exploit what RDS provides 😸:

> enabled_cloudwatch_logs_exports = ["postgresql", "upgrade"]
create_cloudwatch_log_group     = true

For activating backup:
>  backup_window                   = "05:00-05:30"
backup_retention_period = 1

Reply:
Just checked all suggested  variables are either 'null' or 'false' by default.